Log/logger.txt

```diff
@@ -0,0 +1 @@
+[INFO] at:2024-08-15T16:53:40.765750200+03:30 =>: application starting...
```

Comment:
Why we having this file in history?

Reply:
this is test for instance of now date let me fix that

Reply:
commit new one